frontend/types/index.ts

```diff
@@ -130,35 +193,18 @@ export interface IIndexedBuild {
   slug: string
   owner: { username: string }
   latest_game_version: string
-  icons: IIndexedBuildIcon[]
+  icons: IIcon[]
   tags: string[]
   created_at: string
   updated_at: string
-  _links: {
-    cover: {
-      href: string
-      width: number
-      height: number
-    }
-    self: {
-      href: string
-    }
-    versions: {
-      href: string
-    }
-    "add-version": {
-      href: string
-      method: "post"
-    }
-    "toggle-favorite": {
-      href: string
-      method: "post"
-    }
-    followers: {
-      href: string
-      count: number
-    }
-  }
+  _links: IBuildLinks
+}
+
+// IOwner vs IUser??
```

Comment:
There are two user models, `thin` and `full`: [UserModel.cs](backend/src/FactorioTech.Api/ViewModels/UserModel.cs). The `id` is not included in any model because I think at this point you don't need it. So the frontend only knows the id (`sub`) of the logged in user.

Reply:
cool will fix in a following PR with a big rehaul of the types